### DISCUSSION TITLE
Deleted

### DISCUSSION THREAD

**Discussion by 8MiYile** [vernesong/OpenClash#3054] [Q&A]
Deleted

**Comment by evan618** [comment]
别到处喷，人生有不满意的地方自己想办法解决，没有人欠你什么。 本身是个人开发者开发出来的插件，存在BUG 和不合理的地方也无可厚非。换句笃定一点话我认为它就应该有问题，需要大家一起维护和解决，而不是你这种没有贡献的人用着别人的东西，还要骂人家东西不好，用不了就别用，自己想别的办法去。有问题发issues虚心请多学习，自己多google  baidu  bing 。最后这是喂给你的奶：

https://raw.githubusercontent.com/vernesong/OpenClash/core/master/dev/clash-linux-amd64.tar.gz

https://raw.githubusercontent.com/vernesong/OpenClash/core/master/premium/clash-linux-amd64-2023.03.04-5-g4a8cefb.gz

https://raw.githubusercontent.com/vernesong/OpenClash/core/master/meta/clash-linux-amd64.tar.gz
